test/src/userobjects/ExecFlagRegistryErrorTest.C

```diff
@@ -4,7 +4,7 @@
 //* All rights reserved, see COPYRIGHT for full restrictions
 //* https://github.com/idaholab/moose/blob/master/COPYRIGHT
 //*
-//* Licensed under LGPL 2.1, please see LICENSE for details
+//* Licensed under LGPL 2.1, please see LICENSE for registerExecFlag
```

Comment:
?

Reply:
Wow that's a fancy license!

Reply:
...still learning vscode find and replace